torchreid/integration/sc/openvino_task.py

```diff
@@ -197,7 +196,7 @@ def infer(self, dataset: DatasetEntity,
         for i, dataset_item in enumerate(dataset, 1):
             predicted_scene = self.inferencer.predict(dataset_item.numpy)
             dataset_item.append_annotations(predicted_scene.annotations)
-            dataset_item.append_labels(dataset_item.annotation_scene.annotations[0].get_labels())
+            #dataset_item.append_labels(dataset_item.annotation_scene.annotations[0].get_labels())
```

Comment:
@sovrasov @kprokofi why did you add this line, look like it doubles the prediction...

Reply:
Let's try
```python
            predicted_scene = self.inferencer.predict(dataset_item.numpy)
            dataset_item.append_labels(predicted_scene.annotations[0].get_labels())
```